boards/arm/gd32f450i_eval/gd32f450i_eval.dts

```diff
@@ -90,3 +90,9 @@
 	pinctrl-0 = <&dac_default>;
 	pinctrl-names = "default";
 };
+
+&i2c0 {
+	status = "okay";
+	pinctrl-0 = <&i2c0_default>;
+	pinctrl-names = "default";
```

Comment:
as there's an existing EEPROM driver for the device in this board, you can add:
```
&i2c0 {
	status = "okay";
	pinctrl-0 = <&i2c0_default>;
	pinctrl-names = "default";
        clock-frequency = <I2C_BITRATE_FAST>;

	eeprom0: eeprom@50 {
		compatible = "atmel,at24";
		reg = <0x50>;
                ...
	};
};
```

(and alias `eeprom-0 = <&eeprom0>` to use the eeprom sample).

Reply:
done.